aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -315,5 +315,32 @@ Tensor& matmul_out(Tensor &result, const Tensor & tensor1, const Tensor & tensor
   return result;
 }
 
+Tensor frobenius_norm(const Tensor& self) {
+  IntList dim = {0, 1};
+  return at::native::frobenius_norm(self, dim);
 }
+
+Tensor frobenius_norm(const Tensor& self, IntList dim, bool keepdim) {
+  AT_CHECK(
+      dim.size() <= 2,
+      "Improper number of chosend dimensions to do frobenius norm.");
+  if (dim.size() == 0) {
+    Tensor result = self.type().tensor();
+    return at::native::norm_out(result, self, 2, self.dim(), keepdim);
+  }
+  if (dim.size() == 1) {
+    return at::native::norm(self, 2, dim[0], keepdim);
+  }
+  Tensor result = self.type().tensor();
+  return at::sqrt(at::native::sum(self * self, dim, keepdim));
 }
+
+Tensor nuclear_norm(const Tensor& self) {
+  AT_CHECK(
```

Comment:
If we support batch frobenius norm, why do we not support nuclear norm? After all, they should eventually will be called from the same python function.

Reply:
There is one caveat, `gesvd` is not implemented for batched tensors. But I agree, we must have something like `numpy.linalg.norm()` for all of this.

Reply:
The svd function don't support batched tensors now. This PR intends to create the matrix norms only. Extending the function to batched matrix norm is what we want to do in the future.  We may have another PR to let the nuclear norm to support batched norm.